website/content/docs/configuration/keyring/transit.mdx

```diff
@@ -0,0 +1,129 @@
+  outside of Vault, such as Vault Agent.
+
+The following parameters can be set in the `keyring` block but will fallback to
+the values set in the server's [`vault`][vault_config] block if not set
+here. Fields indicated as required must be set either here or the `vault` block.
+
+- `address` `(string: <required>)`: The full address to the Vault cluster.  This
+  may also be specified by the `VAULT_ADDR` environment variable.
+
+- `token` `(string: <required>)`: The Vault token to use. This may also be
+  specified by the `VAULT_TOKEN` environment variable.
+
+- `namespace` `(string: "")`: The namespace path to the transit secret engine.
+  This may also be supplied using the `VAULT_NAMESPACE` environment variable.
+
+- `tls_ca_cert` `(string: "")`: Specifies the path to the CA certificate file used
+  for communication with the Vault server. This may also be specified using the
+  `VAULT_CACERT` environment variable.
+
+- `tls_client_cert` `(string: "")`: Specifies the path to the client certificate
+  for communication with the Vault server. This may also be specified using the
+  `VAULT_CLIENT_CERT` environment variable.
+
+- `tls_client_key` `(string: "")`: Specifies the path to the private key for
+  communication with the Vault server. This may also be specified using the
+  `VAULT_CLIENT_KEY` environment variable.
+
+- `tls_server_name` `(string: "")`: Name to use as the SNI host when connecting
+  to the Vault server via TLS. This may also be specified via the
+  `VAULT_TLS_SERVER_NAME` environment variable.
+
+- `tls_skip_verify` `(bool: "false")`: Disable verification of TLS certificates.
+  Using this option is highly discouraged and decreases the security of data
+  transmissions to and from the Vault server. This may also be specified using the
+  `VAULT_SKIP_VERIFY` environment variable.
+
+## Authentication
+
+Authentication-related values must be provided, either as environment variables
+or as configuration parameters.
+
+~> **Note:** Although the configuration file allows you to pass in `VAULT_TOKEN`
+as part of the keyring's parameters, it is _strongly_ recommended to set these
+values via environment variables.
+
+The Vault token used to authenticate needs the following permissions on the
+transit key:
+
+```hcl
+path "<mount path>/encrypt/<key name>" {
+  capabilities = ["update"]
+}
+
+path "<mount path>/decrypt/<key name>" {
+  capabilities = ["update"]
+}
+```
+
+Other considerations for the token used:
+* It should probably be an [orphan token][], otherwise when the parent token
+  expires or gets revoked the keyring will break.
+* Consider making it a [periodic token][] and not setting an explicit max TTL,
+  otherwise at some point it will cease to be renewable.
+
+## Key rotation
+
+This keyring supports key rotation using the Transit Secret Engine's key rotation
+endpoints. See [doc][rotate_doc]. Old keys must not be disabled or deleted and
```

Comment:
```suggestion
endpoints. Refer to [Vault docs][rotate_doc]. Old keys must not be disabled or deleted and
```

Style guide: use "refer to" instead of "see"